internal/command/action_test.go

```diff
@@ -105,3 +107,27 @@ func TestReturnStatusOfError(t *testing.T) {
 		So(err, ShouldEqual, expected)
 	})
 }
+
+func TestPrintProviderVersions(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping long-running test.")
+	}
+
+	Convey("printProviderVersions should return a string of all of the providers in the DefaultPath", t, func() {
+		pathTo,_ := exec.Command("pwd").Output()
+		pathString := fmt.Sprintf("%s",pathTo)
+		pathString = strings.TrimSuffix(pathString, "\n")
+		pathString = strings.TrimSuffix(pathString, "\r")
+		pathToTest := path.Join(string(pathString), "testversions")
+
+		//test1 - regular formating and appending of version # to string
+		//test2 - chopping off of trailing newline
+		//test3 - failed `--version` call
+		output, err := printProviderVersions(pathToTest)
+		So(err, ShouldBeNil)
+
+		expected := "testprovider1 version 1.2.3\ntestprovider2 version 3.2.1\ntestprovider3: unknown version\n"
+
+		So(output, ShouldEqual, expected)
+	})
```

Comment:
I think there's a test missing when only a single provider path is linked

Reply:
No, I talked to @izgeri and that is going to be a separate card